CausalEstimate/vis/plotting.py

```
@@ -444,20 +446,21 @@ def plot_love(
     threshold: float = 0.1,
     fig: plt.Figure = None,
     ax: plt.Axes = None,
-    figsize: tuple = (8, 10),
+    figsize: tuple = None,
```

Ruff (0.16.2)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)